lib/game_client/ogs/ogs_game.dart

```diff
@@ -47,13 +59,79 @@ class OGSGame extends Game {
   Future<void> manualCounting() => Future.value();
 
   @override
-  Future<void> move(wq.Move move) => Future.value();
+  Future<void> move(wq.Move move) {
+    final sgfMove = move.p.toSgf();
+    return _sendMove(sgfMove, move.col);
+  }
 
   @override
-  Stream<wq.Move?> moves() => Stream.empty();
+  Future<void> pass() {
+    return _sendMove('', myColor);
+  }
+
+  Future<void> _sendMove(String sgfMove, wq.Color color) {
+    // Note: OGS Websocket supports 2-way communication if one appends a requestId
```

Comment:
this would probably be needed. But feel free to do it later on.

Reply:
hmm, that's very odd, thanks for the context. No worries then, let's live with it for now.